zebra-chain/src/transaction/shielded_data.rs

```diff
@@ -121,8 +135,15 @@ impl Arbitrary for Output {
     type Strategy = BoxedStrategy<Self>;
 }
 
+#[derive(Deserialize, Serialize)]
+#[serde(remote = "Either")]
+enum LocalEither<A, B> {
```

Comment:
`LocalEither` seems fine but it would be good to be consistent with whatever naming scheme was used above (previous comment)